xLights/controllers/FPPConnectDialog.cpp

```diff
@@ -1112,10 +1112,21 @@ void FPPConnectDialog::doUpload(FPPUploadProgressDialog *prgs, std::vector<bool>
         }
         row++;
     }
+
+    xLightsFrame* xlframe = static_cast<xLightsFrame*>(GetParent());
     if (messages != "") {
         wxMessageBox(messages, "Problems Uploading", wxOK | wxCENTRE, this);
+        xlframe->SetStatusText("FPP Connect Upload had errors or warnings", 0);
```

Comment:
I would put the messages string at the end of the status text

Reply:
I get your idea, but the messages variable has new line characters and is of unknown length, which doesn't make sense for status bar text.

Instead, I ensured the messages were logged, some of which were already logged, but this puts the contents of the message box together as shown in the message box.